### ISSUE TITLE
Error message when clicking links "Release Notes" and "Executables list" in ReadMe.md

### ISSUE THREAD

**Issue by NeilJustice** [bmatzelle/gow#256]
In file ReadMe.md the following two links currently result in error message "We didn't receive a proper request from your browser. Please contact us if the problem persists."

Release Notes:
http://wiki.github.com/bmatzelle/gow/change_log

Executables list:
http://wiki.github.com/bmatzelle/gow/executables_list

**Comment by bmatzelle** [comment]
Ah thanks.  Apparently Github got rid of their wiki.github.com sub-domain.  I'll update the links.

**Comment by NeilJustice** [comment]
Hi Brent,

Not the world's most pressing issue but I am still seeing error messages when clicking those two links.

Best regards,
Neil

**Comment by bmatzelle** [comment]
Yep.  I updated the Wiki link on the home page but not the readme file.  I'll get to it.  Thanks!
